ceph_deploy/hosts/centos/install.py

```diff
@@ -5,13 +5,13 @@
 from ceph_deploy.util.paths import gpg
 from ceph_deploy.util import net
 
-
 LOG = logging.getLogger(__name__)
 NON_SPLIT_PACKAGES = ['ceph-osd', 'ceph-mon', 'ceph-mds']
 
 
 def rpm_dist(distro):
-    if distro.normalized_name in ['redhat', 'centos', 'scientific', 'oracle', 'virtuozzo'] and distro.normalized_release.int_major >= 6:
+    if distro.normalized_name in ['redhat', 'centos', 'scientific', 'oracle', 'virtuozzo', 'eurolinux'] \
```

Comment:
Maybe we can take the opportunity to do something like this instead:

```python
names = {'redhat', 'centos', 'scientific', 'oracle', 'virtuozzo', 'eurolinux'}
if distro.normalized_name in names and distro.normalized_release.int_major >= 6:
```

Note: it's just a suggestion